### PR TITLE
Update cubecraft

### PR DESCRIPTION
**Pull request by superblaubeere27**

### CODE REVIEW

**Comment by CzechHek** [comment]
CubeCore

**Comment by icewormy3** [comment]
@superblaubeere27 you can either use @CzechHek's cubecraft settings or make your own. Dont be like Senk Ju that 100% relies on the cubecraft autosettings

**Comment by superblaubeere27** [comment]
@icewormy3 Clients with autosettings which are autobanning without telling the user how to use them are trash. This is the best way to waste the users alts and to piss

File: LiquidBounce/autosettings/cubecraft [hunk 1]
```diff
@@ -19,7 +19,7 @@ Criticals Mode HypixelPacket
 FastClimb Mode Normal
 FastClimb NormalSpeed 0.28
 FastUse Mode NCP
-Fly Mode CubeCraft
+Fly Mode NCP
```
Comment (mrphax0r) [inline LiquidBounce/autosettings/cubecraft]: Why? NCP Mode does not bypass afaik

File: LiquidBounce/autosettings/cubecraft [hunk 2]
```diff
@@ -44,10 +44,10 @@ KillAura KeepSprint true
 KillAura LimitedMultiTargets 0
 KillAura LivingRayCast false
 KillAura Mark true
-KillAura MaxCPS 20
+KillAura MaxCPS 10
```
Comment (mrphax0r) [inline LiquidBounce/autosettings/cubecraft]: Hitting with constant 10cps isn't that bypassing

**Comment by superblaubeere27** [comment]
@mrphax0r Yes but it doesn't bypass tho.

**Comment by icewormy3** [comment]
the killaura settings in cubecraft autosettings would get u insta ban because the range is 6 blocks

**Comment by superblaubeere27** [comment]
Nope. I tried it. For hours and I didn't get banned. 6 Range CC Aura works. 6 Range Through Blocks doesn't. Try it before writing a comment

**Comment by Marco09MC** [comment]
https://github.com/CCBlueX/FileCloud/pull/163

**Pull request closed without merging**